### PR TITLE
ErrorCode function updated

### PR DESCRIPTION
**Pull request by adnan1naeem**
added values of form as a param in callback of Error function

### CODE REVIEW

File: src/JsonGraphqlForm/JsonGraphqlForm.tsx [hunk 1]
```diff
@@ -14,7 +14,7 @@ const Query = getComponent<QueryProps>('GraphqlQuery');
 export type mapFormValuesToMutationVariablesFn<Values = any> = (values: Values) => any;
 export type mapQueryDataToInitialValuesFn<Values = any> = (data: any) => Values;
 
-export type JsonGraphqlFormOnErrorFn = (error: ApolloError) => void;
+export type JsonGraphqlFormOnErrorFn<Values = FormikValues> = (error: ApolloError, values: Values) => void;
```
Comment (artalat) [inline src/JsonGraphqlForm/JsonGraphqlForm.tsx]: can you make this interface similar to `JsonGraphqlFormOnSuccessFn`? By adding `actions` param as well? Does this make sense?

**Review by artalat — CHANGES_REQUESTED** [review]

File: src/JsonGraphqlForm/JsonGraphqlForm.tsx (hunk 1 above)
Comment (adnan1naeem) [reply src/JsonGraphqlForm/JsonGraphqlForm.tsx]: the actions param has been added  in Error callback as success callback

File: src/JsonGraphqlForm/JsonGraphqlForm.tsx [hunk 2]
```diff
@@ -14,7 +14,10 @@ const Query = getComponent<QueryProps>('GraphqlQuery');
 export type mapFormValuesToMutationVariablesFn<Values = any> = (values: Values) => any;
 export type mapQueryDataToInitialValuesFn<Values = any> = (data: any) => Values;
 
-export type JsonGraphqlFormOnErrorFn = (error: ApolloError) => void;
+export type JsonGraphqlFormOnErrorFn<Values = FormikValues> = (error: ApolloError, values: Values,
+	actions: FormikActions<Values>
```
Comment (artalat) [inline src/JsonGraphqlForm/JsonGraphqlForm.tsx:20]: please fix formatting

**Review by artalat — CHANGES_REQUESTED** [review]

File: src/JsonGraphqlForm/JsonGraphqlForm.tsx (hunk 2 above)
Comment (adnan1naeem) [reply src/JsonGraphqlForm/JsonGraphqlForm.tsx:20]: fixed it

**Review by artalat — APPROVED** [review]

**Pull request merged**